admin/securedrop_admin/__init__.py

```diff
@@ -85,7 +85,7 @@ def lookup_fqdn(self, fqdn, dns=None):
                 cmd = 'torify ' + cmd
             cmd += ' ' + (dns and dns or '8.8.8.8')
             try:
-                result = subprocess.check_output(cmd, shell=True,
+                result = subprocess.check_output(cmd.split(' '),
```

Comment:
Not going to block on this because what you've done is a strict improvement, but maybe we should programmatically construct a list and then validate input instead of splitting on spaces like this. The former feels more robust.